### PR TITLE
Transform into a `Polars` extension

### PR DESCRIPTION
**Pull request by YuRiTan**
Just an idea. Maybe a nice other way of calling the unpack method?

### CODE REVIEW

File: polars_unpack/README.md
Comment (carnarez) [inline polars_unpack/README.md:1]: Documentation needs updating.

File: polars_unpack/unpack.py
Comment (carnarez) [inline polars_unpack/unpack.py:1]: Some docstrings need updating.

File: polars_unpack/unpack.py [hunk 1]
```diff
@@ -822,6 +760,85 @@ class UnknownDataTypeError(Exception):
     """When an unknown/unsupported datatype is encountered."""
 
 
+@pl.api.register_dataframe_namespace("json")
+@pl.api.register_lazyframe_namespace("json")
```
Comment (carnarez) [inline polars_unpack/unpack.py:767]: Decided to go with the decorators, it feels cleaner.

File: polars_unpack/unpack.py
Comment (carnarez) [inline polars_unpack/unpack.py:1]: I think we should make the `JSON_PATH_SEPARATOR` a parameter somewhere.

File: polars_unpack/unpack.py (hunk 1 above)
Comment (YuRiTan) [reply polars_unpack/unpack.py:767]: Definitely. For some reason I thought I needed to ditch it due to the double api registration xD

**Comment by YuRiTan** [comment]
Good stuff man!

File: pyproject.toml
Comment (YuRiTan) [inline pyproject.toml:1]: Might be nice to package it (in one way or another) right?

Comment (carnarez) [reply pyproject.toml:1]: A `setup.py` file is good enough. I find these `pyproject.toml` MESSY as it gets. Until it gets mandatory...

File: polars_unpack/unpack.py
Comment (carnarez) [reply polars_unpack/unpack.py:1]: Done

Comment (carnarez) [reply polars_unpack/unpack.py:1]: Done

File: polars_unpack/README.md
Comment (carnarez) [reply polars_unpack/README.md:1]: Done

**Review by carnarez — APPROVED** [review]
@YuRiTan let's merge this! 🚀

**Pull request merged**